### ISSUE TITLE
NaN values were obtained during loss.backward

### ISSUE THREAD

**Issue by anvenle** [ziweny11/PhotoRegCodes#7]
Thank you for your work. When I fed the demo data into code, I got an error in loss.backward
![微信图片_20241114231425](https://github.com/user-attachments/assets/cf67a368-d777-45d3-a277-9cc3104a61b1)

**Comment by ziweny11** [comment]
Thank you for your interest.

A similar issue has been addressed previously in issue #5. You can check if you use the latest `extract_RT`. I would be happy to help if you have any further questions.

**Comment by anvenle** [comment]
> Thank you for your interest.
> 
> A similar issue has been addressed previously in issue #5. You can check if you use the latest `extract_RT`. I would be happy to help if you have any further questions.

the same error persists， my code is from two days ago

**Comment by ziweny11** [comment]
The demo code run good on my local setup. Can you print out R2 in function finetuning?
